lib/purescript-cst/src/Language/PureScript/CST/Parser.y

```diff
@@ -147,7 +147,7 @@ many(a) :: { NE.NonEmpty a }
   : many1(a) { NE.reverse $1 }
```

Comment:
This should take care of your last conflict:

```suggestion
  : many1(a) %shift { NE.reverse $1 }
```

Reply:
It did. Thanks!